requirements/requirements_dev.txt

```diff
@@ -1,7 +1,10 @@
 ansible  # Used in build process to generate some configs
 build
-django-extensions
+Django>=4.2.5,<4.3.0
```

Comment:
Can you expand on your comment around this? We don't typically run django management commands in the dev env directly. Usually we do it through a running gateway container.